chroma-manager/tests/integration/installation_and_upgrade/test_update_with_yum.py

```diff
@@ -1,10 +1,11 @@
+from subprocess import check_output
```

Comment:
You don't need this any more.  I'm surprised the pre-commit hook didn't flag this.

Reply:
Just saw this, i'll remove.